packages/react-components/react-positioning/src/middleware/maxSize.test.ts

```diff
@@ -0,0 +1,29 @@
+import { Platform, computePosition } from '@floating-ui/dom';
+import { resetMaxSize } from './maxSize';
+
+const referenceRect = { x: 0, y: 0, width: 100, height: 100 };
+const floatingRect = { x: 0, y: 0, width: 50, height: 50 };
```

Comment:
```suggestion
const referenceRect: TYPE = { x: 0, y: 0, width: 100, height: 100 };
const floatingRect: TYPE = { x: 0, y: 0, width: 50, height: 50 };
```

Can you please type these?

Reply:
typed with `Partial<DOMRect >`